packages/infra/lib/api-stack.ts

```diff
@@ -983,6 +1000,62 @@ export class APIStack extends Stack {
     return documentDownloaderLambda;
   }
 
+  private setupFhirToMedicalRecordLambda(ownProps: {
+    lambdaLayers: lambda.ILayerVersion[];
+    vpc: ec2.IVpc;
+    convertDocLambdaName: string;
+    converterUrl: string;
+    dynamoDBSidechainKeysTable: dynamodb.Table | undefined;
+    bucketName: string | undefined;
+    envType: string;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const {
+      lambdaLayers,
+      vpc,
+      convertDocLambdaName,
+      dynamoDBSidechainKeysTable,
+      converterUrl,
+      bucketName,
+      sentryDsn,
+      envType,
+      alarmAction,
+    } = ownProps;
+
+    const lambdaTimeout = MAXIMUM_LAMBDA_TIMEOUT.minus(Duration.seconds(5));
+    const axiosTimeout = lambdaTimeout.minus(Duration.seconds(5));
+
+    const fhirToMedicalRecordLambda = createLambda({
+      stack: this,
+      name: "FhirToMedicalRecord",
+      runtime: lambda.Runtime.NODEJS_16_X,
+      entry: "fhir-to-medical-record",
+      envVars: {
+        ENV_TYPE: envType,
+        AXIOS_TIMEOUT_SECONDS: axiosTimeout.toSeconds().toString(),
+        CONVERT_DOC_LAMBDA_NAME: convertDocLambdaName,
+        FHIR_TO_CDA_CONVERTER_URL: converterUrl,
+        FHIR_T0_CDA_CONVERTER_KEYS_TABLE_NAME: dynamoDBSidechainKeysTable?.tableName ?? "",
```

Comment:
this should probably be `SIDECHAIN_FHIR_CONVERTER_KEYS_TABLE_NAME` so the sidechain api key code can be re-used in both lambdas